src/Compilers/CSharp/Test/Semantic/CSharpCompilerSemanticTest.csproj

```diff
@@ -47,8 +55,10 @@
       <Name>PdbUtilities</Name>
     </ProjectReference>
   </ItemGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' "></PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' "></PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+  </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|ARM' ">
```

Comment:
@jardpar and @tannergooding: The project system (or at least the old ones) look for a property group with the the $(Configuration)|$(Platform) syntax as a way to enumerate "what are the expected project configurations this can build with". If you don't have these, then it might end up updating the solution file to point to some other configurations, or to something else that you don't expect. Put another way, if you go into project properties and you see a list of the configurations for your project to choose from, this is where they come from.

Assuming you only build one platform, then you should just have the Debug/Release choices for that platform that also matches the default set in the main <PropertyGroup>. If you actually do use some multiple platforms, then obviously have them all.

And yes, there is some sensitivity to the exact syntax here, as it looks at the MSBuild syntax and not just the evaluated result. So don't get too fancy with it.